deny.toml

```diff
@@ -133,6 +133,7 @@ exceptions = [
   # Each entry is the crate and version constraint, and its specific allow
   # list
   #{ allow = ["Zlib"], name = "adler32", version = "*" },
+  { allow = ["MPL-2.0"], name = "webpki-roots", version = "*" },
```

Comment:
This crate uses the `MPL-2.0` license, and thats considered copy-left but we are allowed to distribute executables with it still so its going to be fine for any of our use cases